src/Classifier.jsx

```diff
@@ -233,6 +221,26 @@ class Classifier extends React.Component {
     }
   }
 
+  click_location_in_tile(tile_source, point, context, zoom_level) {
+    // we have to ask the tile source for all the info, then build the tile ourselves. Boo!
+    const tile_vector = tile_source.getTileAtPoint(zoom_level, point);
+    const tile_url = tile_source.getTileUrl(zoom_level, tile_vector.x, tile_vector.y);
+    const tile_bounds = tile_source.getTileBounds(zoom_level, tile_vector.x, tile_vector.y, false);
+    const tile_exists = tile_source.tileExists(zoom_level, tile_vector.x, tile_vector.y);
+    const source_bounds = tile_source.getTileBounds(zoom_level, tile_vector.x, tile_vector.y, true);
+    const pixel_in_tile = new OpenSeadragon.Point();
+    pixel_in_tile.x = Math.floor((point.x - tile_bounds.x) *
+      source_bounds.width /
+      tile_bounds.width);
+    pixel_in_tile.y = Math.floor((point.y - tile_bounds.y) *
+      source_bounds.height /
+      tile_bounds.height);
+    return {
+      tile: new OpenSeadragon.Tile(zoom_level, tile_vector.x, tile_vector.y, tile_bounds, tile_exists, tile_url, context, false, null, source_bounds),
```

Comment:
nice :) this looks much better than what I had. I was under the impression you had to manually load the tile.